src/main/java/org/broadleafcommerce/menu/domain/MenuImpl.java

```diff
@@ -53,7 +53,9 @@
 
 @Entity
 @Inheritance(strategy = InheritanceType.JOINED)
-@Table(name = "BLC_CMS_MENU")
+@Table(name = "BLC_CMS_MENU", indexes = {
+        @javax.persistence.Index(name = "IDX_MENU_NAME", columnList = "NAME")
```

Comment:
Import above and use Index. Also we have moved to jakarta so this needs to be updated